model/ticket.php

```diff
@@ -50,11 +52,29 @@ public function __construct($id)
         $result = $stmt->execute();
 
         if (!$result) return;
-        $result = $stmt->get_result();
+        $result = $stmt->get_result;
         $this->comments = array();
         while ($commentID = $result->fetch_row()) {
             $this->comments[] = new Comment($commentID);
         }
+        //
+        $stmt = DatabaseConnection::getInstance()->prepare(
+            "SELECT attachment.id FROM attachment WHERE attachment.ticket_id=?"
+        );
+        $stmt->bind_param('i', $id);
+        $result = $stmt->execute();
+
+        if (!$result) return;
+        $result = $stmt->get_result;
+        $this->attachments[] = array();
```

Comment:
There's an inconsistency between here, the name of the property in the class and the name of the array appended to on line `71`. *Please* use names consistently, unlike most languages, `PHP` will not warn you if you assign to an undefined variable, so please keep an eye out for mistakes.